### ISSUE TITLE
Unix paths

### ISSUE THREAD

**Issue by jonathandturner** [nrc/rls-analysis#19]
There are Unix paths that might cause Windows trouble.  These are in src/raw.rs:

```
        let principle_path = format!("{}/target/rls/{}/save-analysis", path_prefix, target);
        let deps_path = format!("{}/target/rls/{}/deps/save-analysis", path_prefix, target);
        let libs_path = format!("{}/libs/save-analysis", path_prefix);
```

**Comment by Nonius** [comment]
This one looks good in [17f6381cfbf6f6310a986b771407326f4a204dfe](https://github.com/nrc/rls-analysis/blob/17f6381cfbf6f6310a986b771407326f4a204dfe/src/raw.rs#L134-L136).

**Comment by nrc** [comment]
Fixed by #20
